learn-regular-expressions-by-building-a-spam-filter/spamFilter070.js

```diff
@@ -0,0 +1,22 @@
+const isSpam = msg => {
+  const blacklistRegexes = [
+    /viagra/i,
+    /free money/i,
+    /work from home/i,
+    /stock alert/i,
+    /dear friend/i
+  ];
+  /*
+  Great, refactoring is done, now back to business! Let's start handling those M4ngL3D stR1Ng5 ("mangled strings")!
```

Comment:
Try to simplify your instructions and break this down into multiple steps. It would be better to have the user change each key phrase in `blacklistRegexes` themselves. Introducing character classes and instructing them to change "i" in "viagra" and "dear friend" is a good first step.